versioned_docs/version-2.8/integrations-in-rancher/rancher-desktop.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Kubernetes on the Desktop with Rancher Desktop
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/integrations-in-rancher/rancher-desktop"/>
+</head>
+
+Developing and testing cloud-native applications on your desktop requires a set of foundational blocks such as a virtual machine (if on macOS or Windows), a container run time, Kubernetes, popular utilities, etc. Installing these components individually and getting them to work with each other can get pesky at times. Rancher Desktop nicely bundles all these essential building blocks into an easily installable and manageable desktop application that offers the key features below. 
+
+- Simple and easy installation on macOS, Linux and Windows operating systems.
+- A ready to use light weight Kubernetes distribution (K3s) and the ability to pick Kubernetes versions.
+- GUI-based cluster dashboard powered by Rancher to explore your local cluster.
+- Freedom to choose between multiple container engines (dockerd(moby) vs. containerd).
+- Preferences settings to configure the application to fit your needs.
+- Bundled tools required for your container, Kubernetes-based development, operations workflows.
+- Periodic updates to maintain the bundled tools’ versions up to date.
+- Integrates with proven tools/IDEs (VS Code extensions, Skaffold, etc). 
+- Image & Registry access control.
+- Support for Docker extensions.
+
+To learn more about Rancher Desktop, visit https://rancherdesktop.io and read the docs at https://docs.rancherdesktop.io/ 
+
+To install Rancher Desktop on your machine, refer to the [installation guide](https://docs.rancherdesktop.io/getting-started/installation). 
+
+## Trying Rancher on Rancher Desktop 
+
+Rancher Desktop offers all the necessary setup and tools to make it easy for you to try out containerized and Helm-based applications. For example, you can try out Rancher Kubernetes Management platform right on your desktop using Rancher Desktop by following this [How-to guide](https://docs.rancherdesktop.io/how-to-guides/rancher-on-rancher-desktop).
```

Comment:
```suggestion
Rancher Desktop offers the setup and tools you need to easily try out containerized applications. You can get started with the Rancher Kubernetes Management platform using Rancher Desktop, by following this [How-to guide](https://docs.rancherdesktop.io/how-to-guides/rancher-on-rancher-desktop).
```

"Containerized and Helm-based" sounds a little odd, almost like "Helm-based" is intended as a separate category. Maybe "containerized, Helm-based" (no "and") if we need to mention Helm?

Reply:
I think maybe the intention was to list Helm-based applications as an example of a containerized application?

@gunamata can you clarify what you were trying to convey here?

Reply:
I meant Rancher Desktop helps users in easily trying out containerized apps, helm based apps. For example, we can try out a containerized application via `docker run ...` and a helm chart via `helm install ..`. Rancher Desktop provides the setup and tools out of the box to do these things.

So, `containerized, Helm-based` makes better sense in this context I guess?